tasks/init.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../ruby_task_helper/files/task_helper.rb"
+
+require 'net/http'
+require 'openssl'
+
+class HTTPRequest < TaskHelper
+  def task(**opts)
+    method  = opts[:method].capitalize.to_sym
+    uri     = URI.parse(opts[:base_url]) + (opts[:path] || '')
+    headers = opts[:headers] || {}
+    options = opts.slice(:cacert, :cert, :key)
```

Comment:
`slice` doesn't work on Ruby < 2.5. Since this can run on a remote target rather than with Bolt's Ruby, we should aim for better compatibility.

I also ran into an issue with `compact` in the `ruby_task_helper`. That method doesn't work on Ruby < 2.4.